### ISSUE TITLE
IndexError when using dry.py to remove the water molecules

### ISSUE THREAD

**Issue by TANGLi83-star** [ccsb-scripps/AutoDock-Vina#192]
When using the dry.py, I encountered the following error, I have no idea how to solve it, please help me out. Thanks!
```
water grid score results [ map: receptor.W.map ] 
         [ Water  WEAK  ( -0.36 )  +  ]
         [ Water DISPLC ( -0.20 )  D  ]
         [ Water  WEAK  ( -0.34 )  +  ]
         [ Water DISPLC ( -0.20 )  D  ]
         [ Water  WEAK  ( -0.35 )  +  ]
         [ Water DISPLC ( -0.28 )  D  ]
         [ Water DISPLC ( -0.30 )  D  ]
         [ Water DISPLC ( -0.20 )  D  ]
Traceback (most recent call last):
  File "/home/u2601408/mambaforge/pkgs/adfr-suite-1.0-0/bin/dry.py", line 1194, in <module>
    harvest, affinity = getpoints(grid, coord(w), distance = 1.0) # angstroms
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/u2601408/mambaforge/pkgs/adfr-suite-1.0-0/bin/dry.py", line 739, in getpoints
    harvesting.append( data[z,y,x] )
                       ~~~~^^^^^^^
IndexError: index 51 is out of bounds for axis 2 with size 51
```

**Comment by domiko96** [comment]
Hey Tangli83 I encountered the same problem today. This seems to be a bug.
Did you ever solve your problem?

**Comment by diogomart** [comment]
Fixed in #202. Thanks @domiko96 :+1:
